src/Application/Actions/Hooks/StripePayoutUpdate.php

```diff
@@ -26,8 +26,8 @@
  */
 class StripePayoutUpdate extends Stripe
 {
-
     private SlackChannelChatterFactory $chatterFactory;
+    protected string $slackStripeChannel;
```

Comment:
This class isn't designed to be inherited, is it?
```suggestion
    private string $slackStripeChannel;
```